pkg/apiserver/controllers/v1/decisions.go

```diff
@@ -109,11 +115,18 @@ func (c *Controller) DeleteDecisionById(gctx *gin.Context) {
 func (c *Controller) DeleteDecisions(gctx *gin.Context) {
 	var err error
 
-	nbDeleted, err := c.DBClient.SoftDeleteDecisionsWithFilter(gctx.Request.URL.Query())
+	nbDeleted, deletedFromDB, err := c.DBClient.SoftDeleteDecisionsWithFilter(gctx.Request.URL.Query())
 	if err != nil {
 		c.HandleDBErrors(gctx, err)
 		return
 	}
+	//transform deleted decisions to be sendable to capi
+	deletedDecisions, err := FormatDecisions(deletedFromDB, false)
+	if err != nil {
+		log.Warningf("failed to format decisions: %v", err)
+	}
+	c.DecisionDeleteChan <- deletedDecisions
```

Comment:
do we want to wait for this operation since the channel is not buffered

Reply:
same as above